SwanGallery/.github/workflows/build.yml

```diff
@@ -0,0 +1,149 @@
+name: Build
```

Comment:
[TODO/LATER] These pipelines do not run as they are in a subfolder. They also need to be adapted to the existing pipelines already in the `.github` top level folder.